### PR TITLE
fix: handler and parser coverage

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Add coverage to handlers and log parser.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: tests/test_autonomy/test_cli/test_analyse/test_abci/test_log_parser.py [hunk 1]
```diff
@@ -62,6 +70,15 @@
...
 )
 
+EXPECTED_OUTPUT_ERROR = (
+    "agent         period: 0     round: registration_startup                      event: N/A\n"
+    "\n"
+    "ERRORS:\n"
+    "error\n\n"
+)
+
+EXPECTED_OUTPUT_EVENT = "\nERRORS:\n"
```
Comment (Karrenbelt) [inline tests/test_autonomy/test_cli/test_analyse/test_abci/test_log_parser.py:80]: are "ERRORS:\n" and "\nERRORS:\n" detecting different things?

**Review by angrybayblade — APPROVED** [review]
LGTM, except this failure which does not look like has anything to do with your changes so please open an issue for this: 

```
FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestMonitorBehaviour::test_run
FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestIncrementerBehaviour::test_run
```

**Comment by dvilelaf** [comment]
> LGTM, except this failure which does not look like has anything to do with your changes so please open an issue for this:
> 
> ```
> FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestMonitorBehaviour::test_run
> FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestIncrementerBehaviour::test_run
> ```

#1190

**Review by DavidMinarsch — APPROVED** [review]

**Comment by Karrenbelt** [comment]
> > LGTM, except this failure which does not look like has anything to do with your changes so please open an issue for this:
> > ```
> > FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestMonitorBehaviour::test_run
> > FAILED tests/test_skills/test_counter_client/test_behaviours.py::TestIncrementerBehaviour::test_run
> > ```
> 
> #1190

@angrybayblade @dvilelaf don't forget we also have a [document for specifically this](https://docs.google.com/document/d/1f7DrPdTXaLWJ1icTZ1Mt2S3z3ZdzCodrxAG6d1tI3o0/) - I added the them there

**Pull request merged**